sdk/keyvault/keyvault-common/review/keyvault-common.api.md

```diff
@@ -0,0 +1,43 @@
+## API Report File for "@azure/keyvault-common"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { ChallengeCallbacks } from '@azure/core-rest-pipeline';
+
+// @public
+export function createChallengeCallbacks(options?: CreateChallengeCallbacksOptions): ChallengeCallbacks;
+
+// @public
+export interface CreateChallengeCallbacksOptions {
+    disableChallengeResourceVerification?: boolean;
+}
+
+// @public
+export interface ParsedKeyVaultEntityIdentifier {
+    name: string;
+    vaultUrl: string;
+    version?: string;
+}
+
+// @public
+export type ParsedWWWAuthenticate = {
```

Comment:
I wonder if we could simplify this down to something like

```ts
interface ParsedWWWauthenticate {
  authorization?: string;
  authorization_url?: string;
  resource?: string;
  scope?: string;
  tenantId?: string;
}
```

and not need the fancy type math below? Would love to save some exports